client/my-sites/exporter/export-card/index.jsx

```diff
@@ -77,7 +78,9 @@ class ExportCard extends Component {
 						onClickExport={ this.props.exportSelectedItems }
 					/>
 				</FoldableCard>
-				{ this.props.isExporting && <Interval onTick={ fetchStatus } period={ EVERY_SECOND } /> }
+				{/* Don't fetch status for Jetpack exports, they are not async. */}
```

Comment:
Let's group the comment with code. https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#comments

Also note that we in Calypso we only use `/* comment */` for jsdoc. Any other comments should use `// comment`